docs/5min-overview.md

```diff
@@ -5,6 +5,10 @@ slug: /
 title: Alephium Overview
 ---
 
+import UntranslatedPageText from "@site/src/components/UntranslatedPageText";
```

Comment:
will we need to add this string on every `.md` file?

Reply:
i'm ok with that, just to know :) @nop33

Reply:
yes, on every `.md` file **of the English pages**. So basically only in every md file inside `/docs`. It should not exist in any translated file.

Reply:
FYI: I've added it only to 3 pages to test it out

Reply:
Ok great! seems like a great trade-off compared to what used to be, considering it retains optimal UX :purple_heart:

Reply:
Yes, it's not optimal that it needs to be added in every English md file, but that's what I could come up this morning. I need to move my attention to other things, but I didn't want to leave you struggling with this! 😅